torch/nativert/common/Conv.cpp

```diff
@@ -0,0 +1,51 @@
+#include <torch/nativert/common/Conv.h>
+
+#include <cerrno>
+#include <cstdlib>
+#include <string>
+
+namespace torch::nativert {
+
+template <>
+std::optional<int64_t> tryTo<int64_t>(std::string_view symbol) {
+  // TODO Using strtoll for portability. Consider using std::from_chars in the
+  // future.
```

Comment:
nit: would be nice to link out to https://libcxx.llvm.org/Status/Cxx17.html stating that from_chars isn't fully supported until clang 20 (though for integers you could potentially use it now if MSVC versions have support)